BringgSDK.spec.js

```diff
@@ -151,7 +155,85 @@ describe('BringgSDK', function () {
 
   describe('watch', function(){
     describe('order', function(){
-      describe('module callback', function(){
```

Comment:
nice thing will be to wrap all tests of missing params in their own describe block - `describe('missing params', ..)`